src/overlays/actors/ovl_En_Daiku/z_en_daiku.c

```diff
@@ -25,40 +51,317 @@ const ActorInit En_Daiku_InitVars = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_NONE,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_2,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_NONE,
+        OCELEM_ON,
+    },
     { 20, 60, 0, { 0, 0, 0 } },
 };
 
-#endif
+void EnDaiku_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnDaiku* this = THIS;
+
+    this->actor.colChkInfo.mass = MASS_IMMOVABLE;
+    ActorShape_Init(&this->actor.shape, 0.0f, func_800B3FC0, 40.0f);
+    this->actor.targetMode = 0;
+    Collider_InitAndSetCylinder(globalCtx, &this->collider, &this->actor, &sCylinderInit);
+    this->unk_278 = ENDAIKU_GET_FF(&this->actor);
+    if (this->unk_278 == ENDAIKU_PARAMS_FF_3) {
+        this->unk_288 = ENDAIKU_GET_FF00(&this->actor);
+        this->unk_258 = func_8013D648(globalCtx, this->unk_288, 0x3F);
+    } else if (this->unk_278 == ENDAIKU_PARAMS_FF_2) {
+        this->unk_264 = -2000;
+    }
+
+    if (this->unk_278 == ENDAIKU_PARAMS_FF_0) {
+        this->collider.dim.radius = 30;
+        this->collider.dim.height = 60;
+        this->collider.dim.yShift = 0;
+        this->actor.flags |= 0x8000000;
+        if ((gSaveContext.weekEventReg[63] & 0x80) || ((gSaveContext.day == 3) && gSaveContext.isNight)) {
+            Actor_MarkForDeath(&this->actor);
+        }
+    } else if ((gSaveContext.day == 3) && gSaveContext.isNight) {
+        Actor_MarkForDeath(&this->actor);
+    }
+
+    Math_Vec3f_Copy(&this->unk_26C, &this->actor.world.pos);
+    this->unk_280 = this->actor.world.rot.y;
+    this->actor.gravity = -3.0f;
+
+    switch (this->unk_278) {
+        case ENDAIKU_PARAMS_FF_0:
+            this->unk_27E = this->unk_278 * 4 + 4;
+
+        case ENDAIKU_PARAMS_FF_1:
+            SkelAnime_InitSV(globalCtx, &this->skelAnime, &D_0600A850, &D_06002FA0, this->jointTable, this->morphTable,
+                             17);
+            break;
+
+        case ENDAIKU_PARAMS_FF_2:
+            SkelAnime_InitSV(globalCtx, &this->skelAnime, &D_0600A850, &D_0600B690, this->jointTable, this->morphTable,
+                             17);
+            break;
+
+        case ENDAIKU_PARAMS_FF_3:
+            SkelAnime_InitSV(globalCtx, &this->skelAnime, &D_0600A850, &D_06001114, this->jointTable, this->morphTable,
+                             17);
+            break;
+    }
+
+    func_80943820(this);
+}
+
+void EnDaiku_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnDaiku* this = THIS;
+
+    Collider_DestroyCylinder(globalCtx, &this->collider);
+}
+
+void func_8094373C(EnDaiku* this, s32 arg1) {
+    static AnimationHeader* D_809440A4[] = {
+        &D_06002FA0, &D_0600ACD0, &D_0600C92C, &D_06000C44, &D_0600C234,
+        &D_06000600, &D_06001114, &D_0600B690, &D_0600BEAC,
+    };
+
+    static u8 D_809440C8[] = {
+        0, 0, 0, 0, 2, 0, 0, 2, 2,
+    };
```

Comment:
```suggestion
        &D_06002FA0, &D_0600ACD0, &D_0600C92C, &D_06000C44, &D_0600C234,
        &D_06000600, &D_06001114, &D_0600B690, &D_0600BEAC,
    };
    static u8 D_809440C8[] = {
        0, 0, 0, 0, 2, 0, 0, 2, 2,
    };
```
I think I'm going to start commenting on making sure there aren't spaces between static/non-static variables to match the style of OoT RE: https://discord.com/channels/688807550715560050/688851317593997489/873228050521800775
Very minor thing though